solr/solrj/src/resources/apispec/cluster.Commands.json

```diff
@@ -141,6 +141,21 @@
           }
         }
       }
-    }
+    },
+    "set-placement-plugin": {
```

Comment:
@noblepaul do we still need these awful json apispecs if we use the V2 API annotations?

Reply:
NO, we do not need any more apispecs. I'm planning to eliminate the existing ones

Reply:
Can you please point me @noblepaul or @sigram to existing code in which some of the commands on a path (here `/api/cluster`) use annotations and some use apispec?
I haven't found such a mix, and given existing `/api/cluster` commands (`add-role`, `remove-role`, `set-property`, `set-obj-property`) are defined in the apispec json file, that's where I've added the two new ones.

Reply:
Or, put differently (if there's no simple way to use annotations for the two new commands): when the existing 4 commands are migrated, migrating with them the two new ones is likely not going to make the task any harder.
Keeping the new definitions in apispecs would therefore make sense for now for the sake of simplicity.

Reply:
I'll give patch